Apps/phgitlab/gitlab.json

```diff
@@ -0,0 +1,439 @@
+                    "column_name": "status",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "list projects",
+            "identifier": "list_projects",
+            "description": "List all the projects in the GitLab instance",
+            "verbose": "List all the projects in the GitLab instance",
+            "type": "generic",
+            "read_only": true,
+            "parameters": {},
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "status",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.data.*.*.name",
+                    "data_type": "string",
+                    "example_values": [
+                        "project-name"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.*.path",
+                    "data_type": "string",
+                    "example_values": [
+                        "path-to/project-name"
+                    ]
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "list branches",
+            "identifier": "list_branches",
+            "description": "List the branches of a GitLab project",
+            "verbose": "List the branches of a GitLab project",
+            "type": "generic",
+            "read_only": true,
+            "parameters": {
+                "project_id": {
+                    "description": "The ID of the GitLab project",
+                    "data_type": "numeric",
+                    "required": true,
+                    "primary": true,
+                    "contains": [],
+                    "value_list": [],
```

Comment:
Remove `value_list` and `default` which are not used

Reply:
I removed `value_list` and `default` in [this commit](https://github.com/peterbertel/phantom-apps/commit/8159a235f9d415edc12c6442f09df319460868e2).